snuba/datasets/metrics_aggregate_processor.py

```diff
@@ -0,0 +1,137 @@
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Any, Mapping, Optional
+
+from snuba.clickhouse.value_types import (
+    ClickhouseInt,
+    ClickhouseNumArray,
+    ClickhouseUnguardedExpression,
+)
+from snuba.consumers.types import KafkaMessageMetadata
+from snuba.processor import (
+    AggregateInsertBatch,
+    MessageProcessor,
+    ProcessedMessage,
+    _ensure_valid_date,
+)
+
+
+class MetricsAggregateProcessor(MessageProcessor, ABC):
+    GRANULARITIES = [10, 3600, 3600 * 24]
```

Comment:
Could you define constants for the different granularities?

Reply:
Given that this is a constant, I'm wondering why it's more helpful to subdivide it into more constants?

I can if people want too, though